src/api/nodeRpcCall.js

```diff
@@ -1131,11 +1131,21 @@ const getOfficialUserRegistryEntries = async (walletAddress) => {
 const requestCompanyRegistration = async (companyData, walletAddress)  => {
   const api = await getApi();
 
+  const { registryAllowedToEdit, ...restOfData } = companyData;
```

Comment:
Let's refactor whatever is delivering this data. Action should already be dispatched with `editableByRegistrar` as a separate field.